### ISSUE TITLE
When creating a subscription, singular `@context` is returned as a BadRequest.

### ISSUE THREAD

**Issue by jason-fox** [ScorpioBroker/ScorpioBroker#580]
This is a regression since 4.1.11. Testing with Scorpio 4.1.15. 

### Array of contexts - Succeeds.

Try to create a subscription

```console
curl -L 'http://localhost:1026/ngsi-ld/v1/subscriptions/' \
-H 'Content-Type: application/ld+json' \
--data-raw '{
    "description": "Notify me of low feedstock on Farm:001",
    "type": "Subscription",
    "entities": [
        {
            "type": "FillingLevelSensor"
        }
    ],
    "watchedAttributes": [
        "filling"
    ],
    "notification": {
        "endpoint": {
            "uri": "http://tutorial:3000/subscription/low-stock-farm001-ngsild",
            "accept": "application/json"
        }
    },
    "@context": ["http://context/user-context.jsonld"]
}'
```

Subscription is created ✅ 

### Singular context - Fails

Try to create a subscription with a singular `@context` 

```console
curl -L 'http://localhost:1026/ngsi-ld/v1/subscriptions/' \
-H 'Content-Type: application/ld+json' \
--data-raw '{
    "description": "Notify me of low feedstock on Farm:001",
    "type": "Subscription",
    "entities": [
        {
            "type": "FillingLevelSensor"
        }
    ],
    "watchedAttributes": [
        "filling"
    ],
    "notification": {
        "endpoint": {
            "uri": "http://tutorial:3000/subscription/low-stock-farm001-ngsild",
            "accept": "application/json"
        }
    },
    "@context": "http://context/user-context.jsonld"
}'
```

Subscription is not created ❌  - **400 Bad Request**

```json
{
    "type": "https://uri.etsi.org/ngsi-ld/errors/BadRequestData",
    "title": "Bad Request Data.",
    "detail": "Bad Request Data.",
    "status": 400
}
```

Singular context such as `"@context": "http://context/user-context.jsonld"` is valid JSON-LD and used to work with Scorpio 4.1.11 and earlier.

**Comment by ScorpioBroker** [comment]
fixed in repo.

**Comment by ScorpioBroker** [comment]
Fixed in 5.0.0
